content/doc/book/managing/nodes.adoc

```diff
@@ -169,6 +169,14 @@ d6a84df1fc4f45ddc9c6ab34b08f13391983ffffffffffb3488b7d5ac77fbc7
 . Start the scheduled task and again check that the agent is connected
 * Go back to the node's web page in Jenkins.  If everything works then page should say "Agent is connected"
 
+== Installing a Jenkins agent on Windows
+
+In addition to Windows Scheduler, it is possible to install a Jenkins agent on Windows.
```

Comment:
This sentence is a little confusing for me. Without seeing what comes before it, I don't understand the reference to Windows Scheduler. It sounds like the preceding section might describe how to install Windows Scheduler on Windows.

Since users could be reaching this content via a link, or they might have searched for how to install a Jenkins agent on Windows and then come directly here, we can't assume they will have read the previous section. I would suggest we revise this first sentence so that this lead in can stand on its own.

Reply:
updated first line to ensure clarity of statement and guide users who are looking for instruction